docs/products/kafka/howto/viewing-resetting-offset.rst

```diff
@@ -0,0 +1,145 @@
+View and reset consumer group offsets
+============================================
+
+The `open source Apache Kafka code <https://kafka.apache.org/downloads>`_ includes a ``kafka-consumer-groups.sh`` utility enabling you to view and manipulate the state of consumer groups. 
+Before using the tool, few setup steps need to be performed.
+
+Define the configuration file
+-----------------------------
+
+#. Create the Java keystore and truststore for your Aiven for Apache Kafka service using the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`.
+
+#. Create a ``consumer.properties`` properties file pointing to the keystore and truststore files with the following entries:
+
+* ``security.protocol``: security protocol, SSL for the default TLS security settings
```

Comment:
I have a feeling that this set of ssl properties is usuful even outside of the context of this article. They are what is needed to connect through Java application to our Kafka cluster. Does it make sense to add it as a separate article? Or, we'll do it eventually later, we had something similar in Help articles.

Reply:
Yeah it makes sense, because this needs to be done for every Kafka tool willing to connect to our Kafka... moving into a separate file